plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/TransactionLogTail.java

```diff
@@ -146,24 +146,13 @@ public static Optional<List<DeltaLakeTransactionLogEntry>> getEntriesFromJson(lo
             return Optional.of(resultsBuilder.build());
         }
         catch (IOException e) {
-            if (isFileNotFoundException(e)) {
+            if (e instanceof FileNotFoundException) {
                 return Optional.empty();  // end of tail
             }
             throw new IOException(e);
```

Comment:
Pre-existing: I don't see the point of wrapping the thrown `IOException` inside another `IOException`.

Reply:
Addressed in separate commit `Remove unnecessary wrapping of IOException in TransactionLogTail`